posts/20230223/index.qmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "Reviewing a case study about real estate market prediction"
+author: "Athula Pudhiyidath"
+date: "2023-02-23"
+image: "This image will show on the main page (this is optional).jpg"
+description: A discussion about better practices for binary classification.
```

Comment:
Maybe we can elaborate on what better practices mean here.

Reply:
done a bit more elaboration here